src/controllers/resource-store/index.tsx

```diff
@@ -100,9 +126,14 @@ export const actions: Actions = {
         return withExpiresAt;
       }
 
+      cached.accessedAt = getAccessedAt();
```

Comment:
What happens when `isFromSsr` ? Do we get `accessedAt` already properly set or will it be missing? Probably better to change the isFromSsr condition to override cached instead of dispatching `setResourceState` on its own and return early.
Also worth adding a test ensuring ssr provided data gets `accessedAt` set.

Reply:
✅